Renamer/RenamerCustomParams.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using UnityEngine;
+
+namespace Renamer
+{
+    public class RenamerCustomParams : GameParameters.CustomParameterNode
```

Comment:
This CustomParamaterNode requires that `DisplaySection {get {...}}` be implemented (as of 1.3), but I don't see it in your code, however your PR says you were able to generate binaries.  Do you have other code not yet committed?

Reply:
When i updated for 1.4.0 it wouldnt compile with that included.  So I removed it and everything worked.  Then 1.4.1 came out and DisplaySection was required again. I tried reimplementing it but the new recriuts were no longer getting renamed. I havent been able to find the time to figure out why.

Reply:
Ok, so it sounds like we're both on the same page then.  :)  I'll see if I can't figure out what's up.